manageability-and-operations/observability-and-manageability/external-database-enablement/root_module/modules/container/container_connection/variables.tf

```diff
@@ -0,0 +1,106 @@
+#Common database system variables
+
+variable "compartment_ocid" {
+  type = string
+}
+
+variable "external_database_connector_connection_string_hostname" {
+  description = "Hostname for connection string"
+  type        = string
+  validation {
+  	condition = var.external_database_connector_connection_string_hostname != ""
+	error_message = "The value of 'host' in the JSON-input for database systems is an empty string"
+  }
+}
+
+variable "external_database_connector_connection_string_port" {
+  description = "Port for connection string"
+  type        = number
+  default     = 1521
+}
+
+variable "external_database_connector_connection_string_protocol" {
+  description = "Protocol for connection string. Can be 'TCP' or 'TCPS'"
+  type        = string
+  validation {
+    condition     = contains(["TCP", "TCPS"], var.external_database_connector_connection_string_protocol)
+    error_message = "The value of 'protocol' in the JSON-input for database systems must be 'TCP' or 'TCPS'"
+  }
+}
+
+variable "ssl_secret_id" {
+  description = "The OCID for the OCI Vault Secret used for TCPS-connections. PKCS12 and JKS are supported as trust store types"
+  type        = string
+  default     = null
+}
+
+variable "external_database_connector_agent_id" {
+  description = "The OCID for the management agent used for database connections"
+  type        = string
+  validation {
+  	condition = var.external_database_connector_agent_id != ""
+	error_message = "The value of 'managementAgentId' in the JSON-input for database systems is an empty string"
+  }
+  #default = "ocid1.managementagent.oc1..XXXXXXX"
```

Comment:
Remove line.

Reply:
Hi Olaf, are you referring to removing the default value parameters in these files you are mentioning?

Reply:
Specifically the ones where there are placeholder OCIDs?